test/unit/test.cpp

```diff
@@ -44,19 +44,20 @@ TEST_CASE( "Utilities" ) {
 
 
 void check(string parser_name) {
-	auto * loader = get_swidtagio(parser_name.c_str());
+	auto loader = SWIDTagIO();
```

Comment:
wait, won't this construct it on the stack via its ctor and then run the implicit operator= to copy it to another thing on the stack? even if it gets optimized out this is a really weird pattern. Just use:
```
SWIGTagIO loader();
```

Reply:
I think that it would be optimized out, and it is a test anyway, but why not to make it simpler.